var/spack/repos/builtin/packages/py-oslo-config/package.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyOsloConfig(PythonPackage):
+    """
+    The Oslo configuration API supports parsing command line arguments and .ini
+    style configuration files.
+    """
+
+    homepage = "https://docs.openstack.org/oslo.config/"
+    pypi     = "oslo.config/oslo.config-8.7.1.tar.gz"
+
+    maintainers = ['haampie']
+
+    version('8.7.1', sha256='a0c346d778cdc8870ab945e438bea251b5f45fae05d6d99dfe4953cca2277b60')
+
+    depends_on('python@3.6:', type=('build', 'run'))
+    depends_on('py-setuptools', type='build')
```

Comment:
Missing build dep on `py-pbr@2.0.0:`

Reply:
hm yeah, it's not listed in requirements.txt :thinking: just in setup.py